### ISSUE TITLE
ES5 version includes ES6 version and viceversa

### ISSUE THREAD

**Issue by fregante** [SplittyDev/evee.js#5]
The suggested import/require doesn't work as you expect.

```sh
echo "require('evee').Evee;" > index.js
browserify index.js
-> http://pastebin.com/2gdaAVmR
```

browserify's output contains both the ES6 and ES5 versions, so Evee 2 completely fails on non-ES6 browsers currently.

To do what you want, you need two files and not a joint entry point.

I'd suggest this configuration:

* delete index.js
* copy `lib/evee.js` to `es6.js` in prepublish
* package.json:
  ```json
  {
    "main": "dist/evee.js"
  }
  ```

This way you can import the two versions separately:

```js
var Evee = require('evee'); //es5
var Evee = require('evee/es6'); //es6
import Evee from 'evee/es6'; //es6 + modules
```

**Comment by SplittyDev** [comment]
You're right. Thanks for pointing that out!
A fix should be out soon.

**Comment by SplittyDev** [comment]
Hey @bfred-it, thanks again for reporting this issue!
I published a new version and deprecated the older v2.0.* versions.

The new API is exactly as you suggested,
you can import the ES6 version using `evee/es6`.

No idea why I even did that index.js thing in the first place.
I hope everything is working now :)
